packages/graphback-codegen-schema/src/schemaTemplate.ts

```diff
@@ -122,26 +144,24 @@ export const generateSchemaString = (context: TargetContext, customContext?: Cus
   const allQueries = generateQueries(queries, customContext.customQueries)
   const allMutations = generateMutations(mutations, customContext.customMutations)
   const allSubs = generateSubscriptions(subscriptions, customContext.customSubscriptions)
-  let output = `
-  ${nodeInterfaces(interfaces)}
+  let output = `${nodeInterfaces(interfaces)}
 
-  ${nodeTypes(types)}
+${nodeTypes(types)}
 
-  ${inputs(inputFields)}
+${inputs(inputFields)}
 
-  ${filters(filterFields)}
-  `
+${filters(filterFields)}`
 
   if (allQueries) {
-    output += `\n\n  ${allQueries}`
```

Comment:
I think in future thngs like this could happen in separate PR as it increases PR size tremendously and does not hugely contribute towards PR scope.

Reply:
Yes.. This is on me. I tried to pull some things back from this PR however the way I approached this PR was to make things working on top of the template, which sadly involved fixing issues in the schema and client-side generators.